xray/utils/resultwriter.go

```diff
@@ -31,14 +31,20 @@ const maxPossibleCve = 10.0
 
 var OutputFormats = []string{string(Table), string(Json), string(SimpleJson), string(Sarif)}
 
+type SarifProperties struct {
```

Comment:
It looks like this struct can be made private.

Reply:
done